# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
# Result definition
# The result specifies the chosen points for each agent
Point2D[] goals
